src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/DomainEventPublisherTest.kt

```diff
@@ -0,0 +1,76 @@
+package uk.gov.justice.digital.hmpps.hmppsalertsapi.service.event
+
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.entity.event.AlertAdditionalInformation
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.entity.event.AlertDomainEvent
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.enumeration.DomainEventType.ALERT_CREATED
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.enumeration.Source.NOMIS
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.integration.wiremock.PRISON_NUMBER
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.utils.ALERT_CODE_VICTIM
+import uk.gov.justice.digital.hmpps.hmppsalertsapi.utils.testObjectMapper
+import uk.gov.justice.hmpps.sqs.HmppsQueue
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+import java.time.LocalDateTime
+import java.util.UUID
+
+class DomainEventPublisherTest {
+  private val objectMapper = testObjectMapper()
+
+  private val baseUrl = "http://localhost:8080"
+
+  @Test
+  fun `publish alert event`() {
+    val hmppsQueueService = mock<HmppsQueueService>()
```

Comment:
Should these mocks be fields on the class?

Reply:
Good shout. Moved